src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsremandandsentencingapi/jpa/repository/RecallRepository.kt

```diff
@@ -6,4 +6,5 @@ import java.util.UUID
 
 interface RecallRepository : CrudRepository<RecallEntity, Int> {
   fun findOneByRecallUniqueIdentifier(recallUniqueIdentifier: UUID): RecallEntity?
+  fun findByPrisonerId(prisonerId: String): List<RecallEntity>
```

Comment:
Potentially may need to paginate this - just being used fir the POC at the moment so no decision yet